Kickstarter-iOS/Features/PledgeView/Controllers/PostCampaignCheckoutViewController.swift

```diff
@@ -199,6 +205,28 @@ final class PostCampaignCheckoutViewController: UIViewController {
         self.presentHelpWebViewController(with: helpType, presentationStyle: .formSheet)
       }
 
+    self.viewModel.outputs.validateCheckoutSuccess
+      .observeForControllerAction()
+      .observeValues { [weak self] _ in
+        guard let self else { return }
+
+        // TODO: Confirm paymentIntent using Stripe.confirmPayment()
+      }
+
+    self.viewModel.outputs.validateCheckoutExistingCardSuccess
+      .observeForControllerAction()
+      .observeValues { [weak self] _ in
+        guard let self else { return }
+
+        // TODO: Confirm paymentIntent using Stripe.confirmPayment()
```

Comment:
Is this ``// TODO:`` correct for the existing card case, too?

Reply:
Yes we'll have to confirm the payment either way using the intent